#168285248, #168285305 Rest Endpoint accept and reject

Regiss05 opened the pull request:
#### What does this PR do?
accept and deny request

#### Description of Task to be completed?
mentor should accept or deny mentorship request session

#### How should this be manually tested?
- /api/v1/sessions/:sessionId/accept
- /api/v1//sessions/:sessionId/reject

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/168285305
- https://www.pivotaltracker.com/story/show/168285248

#### Screenshots (if appropriate)
![feed end7](https://user-images.githubusercontent.com/38294716/64370171-2fbca800-d01e-11e9-894f-215fc397164b.png)


#### Questions:

Regiss05 committed:
ft(endpoint): signup
user can signup
[finishes #168284762]

Regiss05 committed:
ft(endpoint): signin
user login
[finishes #168284840]

Regiss05 committed:
ft(endpoint): all mentors
user can view all mentors
[finishes #168284997]

Regiss05 committed:
ft(endpoint): specific mentor
user can view specific mentor
[finishes #168284997]

Regiss05 committed:
ft(endpoint): specific mentor
user can view specific mentor
[finishes #168285047]

Regiss05 committed:
ft(endpoint): session request
user should create a mentorship session request
[finishes #168285166]

Regiss05 committed:
ft(endpoint): change mentor to user
mentor can change user to mentor[finishes #168285220]

Regiss05 committed:
ft(endpoint): accept and denied
mentor can accept or decline mentorship request
[finishes #168285248,#168285305]

oesukam commented on .env:
```diff
@@ -1,5 +1,2 @@
-}
\ No newline at end of file
+
+    secret = cycle10
```
Blank line required and .env file should be ignored, you can’t push it to public

Regiss05 replied:
yes I'm going to put it into gitignore

oesukam commented on .nyc_output/defd535a-6e2d-4f09-8402-eb289b987973.json:
```diff
@@ -0,0 +1 @@
+{}
```
blank line required

oesukam commented on .nyc_output/processinfo/defd535a-6e2d-4f09-8402-eb289b987973.json:
```diff
@@ -0,0 +1 @@
+{"uuid":"defd535a-6e2d-4f09-8402-eb289b987973","parent":null,"pid":10388,"argv":["C:\\Program Files\\nodejs\\node.exe","D:\\free-mentors\\node_modules\\mocha\\bin\\mocha","--exit","server/v1/tests/*js"],"execArgv":[],"cwd":"D:\\free-mentors","time":1567686085186,"ppid":19568,"root":"85ea48c0-eb6d-48cd-84b3-425c9eb24638","coverageFilename":"D:\\free-mentors\\.nyc_output\\defd535a-6e2d-4f09-8402-eb ...
```
This file and the entire folder should be ignored

oesukam commented on .travic.yml:
```diff
@@ -0,0 +1,22 @@
+  - skip-cleanup: true
+  - github-token: $GITHUB-TOKEN
+
+  on:
```
Blank line required

Regiss05 committed:
ft(endpoint): accept and denied
mentor can accept or decline mentorship request
[finishes #168285248,#168285305]

Regiss05 committed:
ft(endpoint): accept and denied
mentor can accept or decline mentorship request
[finishes #168285248,#168285305]

Regiss05 committed:
ft(endpoint): accept and denied
mentor can accept or decline mentorship request
[finishes #168285248,#168285305]

Regiss05 committed:
ft(endpoint): accept and denied
mentor can accept or decline mentorship request
[finishes #168285248,#168285305]

Regiss05 committed:
ft(endpoint): accept and denied
mentor can accept or decline mentorship request
[finishes #168285248,#168285305]

oesukam commented on .env:
```diff
@@ -1,5 +1,2 @@
-  }
-}
\ No newline at end of file
+
```
New line required

oesukam commented on .nyc_output/processinfo/index.json:
```diff
@@ -0,0 +1 @@
+{"processes":{"322b87ab-99a9-4a78-b3e7-d81904ba9dba":{"parent":null,"children":[]}},"files":{},"externalIds":{}}
```
This file and folder should be ignore

oesukam reviewed (approved):
Add the screenshots for postman

Regiss05 committed:
ft(endpoint): accept and denied
mentor can accept or decline mentorship request
[finishes #168285248,#168285305]
